tests/tests_accept_eula.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure that the role runs when EULA are accepted
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_2019_standard_eula: true
+  tasks:
+    - name: Verify the failure on a fresh host when required vars undefined
+      block:
+        - name: Run the role with default parameters
+          import_role:
```

Comment:
use `include_role` instead of `import_role`

Reply:
Done. Why so?